src/openpgp.js

```diff
@@ -99,7 +99,7 @@ export function destroyWorker() {
 export function generateKey({ userIds=[], passphrase, numBits=2048, unlocked=false } = {}) {
   const options = formatUserIds({ userIds, passphrase, numBits, unlocked });
 
-  if (!util.getWebCryptoAll() && asyncProxy) { // use web worker if web crypto apis are not supported
+  if (asyncProxy) { // use web worker when available
```

Comment:
you are right, but having a long key generation in a webworker is not an issue for me, while it is having a signing or other stuff blocking the main thread as the application reamins frozen all the time.